src/CPUSolver.cpp

```diff
@@ -135,6 +135,9 @@ void CPUSolver::initializeFluxArrays() {
     size = _num_FSRs * _num_groups;
     _scalar_flux = new FP_PRECISION[size];
     _old_scalar_flux = new FP_PRECISION[size];
+    
+    /* Set the old scalar flux to zero */
+    memset(_old_scalar_flux, 0.0, sizeof(FP_PRECISION) * size);
```

Comment:
The ``Solver::flattenFSRFluxes()`` routine is called immediately after ``initializeFluxArrays()`` so it should have the same effect. Also, as you may have noticed, I avoid using ``memset`` whenever possible since I have observed some irreproducible results when using it - it seems to be *non-blocking* and return before the values are truly initialized in memory, or there can be cache coherency issues, or something else altogether. As a result, I initialize arrays with explicit ``for`` loops as is done in ``Solver::flattenFSRFluxes()``.